Sources/AsyncCollections/Map.swift

```diff
@@ -10,13 +10,20 @@ extension Sequence where Element: Sendable {
     ///     element of this sequence as its parameter and returns a transformed value of
     ///     the same or of a different type.
     /// - Returns: An array containing the transformed elements of this sequence.
-    public func asyncMap<T: Sendable>(_ transform: @Sendable @escaping (Element) async throws -> T) async rethrows -> [T] {
```

Comment:
No need for `T: Sendable` or `@escaping`

Reply:
Not really. I've already enabled strict-concurrency and that's not the case.

Reply:
I get warnings with it enabled. How are you enabling it?

Reply:
You're right about it for the concurrent one though, and i've already reverted that change.

Reply:
opps the usual problem with GitHub not updating comments properly.

> I get warnings with it enabled. How are you enabling it?

I've enabled it for the whole package in Swift 5.9, and already resolved some problems because of that.
I'm doing clean builds, i'll try cleaning derived data as well, or even in a dev container.

Reply:
Xcode 15.1 beta2: no warnings

Reply:
Swift 5.9 & 5.9.1 on Linux, no warnings.
It kind of makes sense too that it doesn't need to be Sendable.